setup.py

```diff
@@ -38,6 +38,7 @@
 from distutils import extension as _extension
 import setuptools
 from setuptools.command import egg_info
+import six
```

Comment:
I'm *slightly* uncomfortable requiring that a user have `six` installed at install invocation time as opposed to it being pulled in by `setuptools`. Mind using a standard lib equivalent to check for Python 3 (or pulling `six` in via a custom command like `commands.BuildExt`)?